includes/admin/meta-boxes/views/html-order-items.php

```diff
@@ -196,26 +200,62 @@
 		<?php do_action( 'woocommerce_admin_order_totals_after_tax', $order->get_id() ); ?>
 
 		<tr>
-			<td class="label"><?php esc_html_e( 'Total', 'woocommerce' ); ?>:</td>
+			<td class="label"><?php esc_html_e( 'Order Total', 'woocommerce' ); ?>:</td>
 			<td width="1%"></td>
 			<td class="total">
-				<?php echo $order->get_formatted_order_total(); // WPCS: XSS ok. ?>
+				<?php echo wc_price( $order->get_total(), array( 'currency' => $order->get_currency() ) ); // WPCS: XSS ok. ?>
 			</td>
 		</tr>
 
-		<?php do_action( 'woocommerce_admin_order_totals_after_total', $order->get_id() ); ?>
+	</table>
+
+	<div class="clear"></div>
+
+	<?php if ( in_array( $order->get_status(), array( 'processing', 'completed', 'refunded' ) ) ) : ?>
+
+		<table class="wc-order-totals" style="border-top: 1px solid #999; margin-top:12px; padding-top:12px">
+			<tr>
+				<td class="label"><?php esc_html_e( 'Paid By Customer', 'woocommerce' ); ?>:</td>
```

Comment:
Maybe this was discussed already, but I wonder if adding this "Paid By Customer" line will confuse the store manager when an offline payment gateway is used? I say this because when a customer selects a payment gateway like "Cash on delivery" during checkout, the order status is set to `processing`, but the customer hasn't paid anything yet.

Reply:
@LevinMedia and @peterfabian, I added commit 44ecb0ec9e509eee91cc2eb69cba892a824271cf to address the issue described above. It basically checks for the presence of the `_date_paid` order meta to decide whether or not to display the "Paid by customer" section. More information on the commit message.

Reply:
Thanks @rodrigoprimo !